app/server.py

```diff
@@ -18,6 +18,46 @@ def index():
 ######################################################################
 @app.route('/shopcarts/<int:id>', methods=['GET'])
 def get_shopcarts(id):
+    """
+    Get a Shopcart
+    This endpoint will get a shopcart.
+    ---
+    tags:
+      - Shopcarts
+    produces:
+      - application/json
+    parameters:
+      - name: user_id
```

Comment:
parameter name doesn't match the actual param. Can we actually change the parameter in the function definition as `user_id` is more explicit.